src/github/client/interceptors.ts

```diff
@@ -96,12 +97,6 @@ export const handleFailedRequest = (logger: Logger) =>
 				return Promise.reject(new BlockedIpError(error, status));
 			}
 
-			if (status === 403) {
-				const rateLimitRemaining: number = parseInt(rateLimitRemainingHeaderValue);
-				logger.warn({ err: error }, "Github returned 403 without ratelimit header");
-				return Promise.reject(new RateLimitingError(Date.now() / 1000 + 60 * 60, rateLimitRemaining, error, status));
-			}
-
```

Comment:
Removed this block, because we don't want to handle all 403s as rate limit events. It may be a permissions issue and this would hide it behind a rate limit error (we had that case in the old code, this is probably where this code here comes from)

Reply:
I would suggest to add some extra logging for that case. I remember in the past we had some sneaky rate limit errors which were masked as 403s but didn't have enough info on them to decide that it is a rate limit.